plugins/tracers/gstsharktracer.c

```diff
@@ -223,6 +226,10 @@ gst_shark_tracer_constructed (GObject * object)
   GstSharkTracer *self = GST_SHARK_TRACER (object);
 
   gst_shark_tracer_save_params (self);
+  if (g_shark_tracer_refcount == 0) {
+    gst_ctf_init ();
+  }
+  g_shark_tracer_refcount++;
```

Comment:
[g_atomic_int_inc](https://developer-old.gnome.org/glib/stable/glib-Atomic-Operations.html#g-atomic-int-inc)

Reply:
For this one I think we need to know the value before incrementing, I used [g_atomic_int_add](https://www.freedesktop.org/software/gstreamer-sdk/data/docs/latest/glib/glib-Atomic-Operations.html#g-atomic-int-add) for this.